cargo-shuttle/src/suggestions.rs

```diff
@@ -0,0 +1,225 @@
+//! Suggestions to be shown to users encountering errors while using cargo-shuttle.
+// TODO: Ideally, the suggestions would be inferred from the status codes returned by
+// the gateway in case of requests to it, or errors thrown by the client doing work
+// on the users machines. This is a naive way of handling the errors that should suggest
+// retrying common commands or reach out on our Discord server in case failures persist.
+
+use crossterm::style::Stylize;
+
+// --------------------------
+// API key related
```

Comment:
Just one trivial comment: instead of separating different contexts in the same file with comments, wouldn't it be better if they have been in different modules? Not a big deal though :) LGTM

Reply:
Done.